### ISSUE TITLE
Pressure Sensitive button support

### ISSUE THREAD

**Issue by Gorilli09** [open-goal/jak-project#2773]
### What feature is your idea related to?

My idea is related to Pressure Sensitive buttons used in the DualShock 2, DualShock 3, and Original Xbox controllers.

### Describe the solution you'd like.

Pressing down on the accelerate/brake buttons lightly make the vehicle go/brake slowly, and pressing the accelerate/brake buttons down firmly make the vehicle go/brake faster.

### Additional context

This could be useful for people who have a DualShock 3.

**Comment by ManDude** [comment]
This feature is outside of our control. This is handled by drivers and other controller APIs which we have no control over.

**Comment by ZippyZolton** [comment]
iirc PCSX2 was able to pull it off, but it probably won't be easy even with help from them

**Comment by xTVaser** [comment]
It doesn't work natively via SDL on windows (but works on other platforms) https://github.com/libsdl-org/SDL/issues/5148
And from a recent (and still unmerged) PR on pcsx2, it doesn't look like it was solved over there either https://github.com/PCSX2/pcsx2/pull/9138

TLDR, it's a giant headache for not a ton of gain -- the newer dualshock controllers don't even support pressure sensitivity.

**Comment by ManDude** [comment]
We're not gonna be implementing our own controller drivers and API so this is not our problem.
